src/libraries/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -232,9 +244,21 @@ public void PathsWithInvalidColons_ThrowIOException_Core(string invalidPath)
             Assert.ThrowsAny<IOException>(() => Create(invalidPath));
         }
 
+        [Theory, InlineData(@"file:///C|/My Documents/ALetter.html")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void PathsWithInvalidPipeChar_ThrowIOException(string invalidPath)
+        {
+            // You can't actually create a directory with a colon in it. It was a preemptive
```

Comment:
This is talking about a colon.  The InlineData is using a bar, not a colon.  Is this a copy/paste error, or am I misreading it?

Reply:
The comment does not belong here. Copy/paste error. I'll remove it.

I copied the `PathsWithInvalidColons_ThrowIOException_Core`method to test the pipe character separately.

I had to remove the string with a pipe from the list in TestData.cs because it was the only case that was failing differently in Framework (`ArgumentException`).